src/utils/violations/altTab.js

```diff
@@ -0,0 +1,16 @@
+import { VIOLATIONS } from '../constants';
+
+export default function detectAltTab(handleViolation) {
```

Comment:
Each file for the shortcut is not necessary. You could've discussed this. PLease move it to one file.

Reply:
resolved